traitlets/config/application.py

```diff
@@ -6,46 +6,45 @@
 
 from __future__ import print_function
 
+from collections import defaultdict, OrderedDict
 from copy import deepcopy
 import json
 import logging
 import os
 import re
 import sys
-from collections import defaultdict, OrderedDict
-
-from decorator import decorator
-
 from traitlets.config.configurable import Configurable, SingletonConfigurable
 from traitlets.config.loader import (
     KVArgParseConfigLoader, PyFileConfigLoader, Config, ArgumentError, ConfigFileNotFound, JSONFileConfigLoader
 )
-
 from traitlets.traitlets import (
     Bool, Unicode, List, Enum, Dict, Instance, TraitError, observe, observe_compat, default,
 )
+
+from decorator import decorator
+from ipython_genutils import py3compat
 from ipython_genutils.importstring import import_item
 from ipython_genutils.text import indent, wrap_paragraphs, dedent
-from ipython_genutils import py3compat
-
 import six
 
+
 #-----------------------------------------------------------------------------
 # Descriptions for the various sections
 #-----------------------------------------------------------------------------
-
 # merge flags&aliases into options
 option_description = """
-Arguments that take values are actually convenience aliases to full
-Configurables, whose aliases are listed on the help line. For more information
-on full configurables, see '--help-all'.
-""".strip() # trim newlines of front and back
+The options below are convenience aliases to configurable class-options,
```

Comment:
Well, my idea was to separate cmd-line *class-options* (e.g. `--Foo.bar`) from source-code *class-parameters* (e.g. the `bar` parameter of `Foo` class). 
So it is correct in this case because the next phrase refers to the "Equivalent to" desc-line, which indeed it contains *options*, not *parameters* (see PR text for comparison).

Another alternative would be to use *class-attributes* instead of *parameters` which is accurate, but I wouldn't dare such an invasive terminology change.